README.md

```diff
@@ -45,6 +45,21 @@ inherit_gem:
 # your customizations here...
 ```
 
+### RSpec
+
+To analyse RSpec files, add the [rubocop-rpec](https://rubygems.org/gems/rubocop-rspec) gem to your Gemfile and in your RuboCop configuration require it alongside the RSpec-specific yaml file:
```

Comment:
[nits] "To lint"?

Reply:
updated ef233f7efbd8846ff7704a2bb10c524ef60503a6